distributed/client.py

```diff
@@ -4294,6 +4295,44 @@ def collections_to_dsk(collections, *args, **kwargs):
         """Convert many collections into a single dask graph, after optimization"""
         return collections_to_dsk(collections, *args, **kwargs)
 
+    async def _story(self, keys=(), on_error="raise"):
+        stimulus_id = f"client-story-{time()}"
+        info = self.scheduler_info()
+
+        msg = {"op": "get_story", "keys": keys}
+        WORKER_SENTINEL = [("worker-story-retrieval-failure", stimulus_id, time())]
+        SERVER_SENTINEL = [("scheduler-story-retrieval-failure", stimulus_id, time())]
+
+        coros = [
+            send_message(
+                msg,
+                self.rpc,
+                self.scheduler.address,
+                None,
+                on_error,
+                default=SERVER_SENTINEL,
+            )
+        ] + [
+            send_message(
+                msg, self.rpc, address, None, on_error, default=WORKER_SENTINEL
+            )
+            for address in info["workers"]
+        ]
```

Comment:
I recommend instead using `client.scheduler.get_story` and `client.scheduler.broadcast(msg)`.  I don't think that we want to reach into `send_message` here.  That API is rarely used and somewhat low-level.  Broadcast is more common for things like this.

Reply:
Also, it's just nicer if we don't need to introduce new ways of doing things.

Reply:
> I recommend instead using client.scheduler.get_story and client.scheduler.broadcast(msg)

Done. Another +1 for this recommendation is that  network topologies probably exist where the client has access to the scheduler, but not the workers.